app/controllers/search_controller.rb

```diff
@@ -1,7 +1,7 @@
 class SearchController < ApplicationController
   def index
     @facade = OomprBeFacade.new
-    @listings = @facade.search_listings_by_city(search_params[:city])
+    @listings = Kaminari.paginate_array(@facade.search_listings_by_city(search_params[:city])).page(params[:page]).per(3)
```

Comment:
There will probably be a merge conflict when my PR with the caching is merged, because I changed this line in order to cache this API call. That's all good, I'll get it figured out!

Reply:
yeah, looks like there's just two conflicts - in the Gemfile and this controller. should be a quick fix tomorrow morning!